pyiron_base/project/archiving/import_archive.py

```diff
@@ -91,6 +81,7 @@ def import_jobs(project_instance, archive_directory, df, compressed=True):
         job_id = pr_import.db.add_item_dict(par_dict=entry, check_duplicates=True)
         job_id_lst.append(job_id)
 
+    # print(job_id_lst)
```

Comment:
```suggestion
```

Reply:
Alternatively, if this is commonly used for debugging, then it might make sense to add it to the log.

Reply:
nice catch I don't know how it appeared